src/System.Windows.Forms/src/System/Windows/Forms/ToolStripPanel.FeedbackRectangle.cs

```
@@ -60,7 +58,7 @@ protected void Dispose(bool disposing)
                     {
                         Visible = false;
                         _dropDown.Dispose();
-                        _dropDown = null;
+                        _dropDown = null!;
```

Perhaps declare this field as nullable, as it's tested for null in `Visible` property.

---

You are right, it should not happen. I guess the reason these checks were added in the past was that someone was accessing the Visible property after the disposal and instead of fixing the caller, we changed the framework to be defensive instead of failing fast. Perhaps this was a Watson report. However, we don't know what the scenario was and if it's still relevant. So the compatible thing to do is to not risk an exception in that property.

---

I added a separate commit that makes `_dropDown` nullable. Please note that this means adding null checks in a couple of places.